src/fides/api/models/privacy_experience.py

```diff
@@ -612,70 +534,3 @@ def upsert_privacy_experiences_after_config_update(
             )
             linked_regions.append(region)
     return linked_regions, unlinked_regions
-
-
-def cache_saved_and_served_on_consent_record(
-    db: Session,
-    consent_record: Union[
-        PrivacyNotice,
-        TCFPurposeConsentRecord,
-        TCFPurposeLegitimateInterestsRecord,
-        TCFSpecialPurposeRecord,
-        TCFFeatureRecord,
-        TCFSpecialFeatureRecord,
-        TCFVendorConsentRecord,
-        TCFVendorLegitimateInterestsRecord,
-        TCFVendorRelationships,
-    ],
-    fides_user_provided_identity: Optional[ProvidedIdentity],
-    record_type: Optional[ConsentRecordType],
```

Comment:
Obsolete functionality removed. 

We no longer support `GET /privacy-exerience?fides_user_device=xxxxx`, supplementing an Experience with user's saved TCF preferences.  This is a time-consuming operation and is only used in the privacy center.  Generally we haven't used this functionality in the way we thought we would.

Ticket to remove dependency in FE https://ethyca.atlassian.net/browse/PROD-1485